llama-index-integrations/readers/llama-index-readers-google/llama_index/readers/google/maps/base.py

```diff
@@ -34,7 +32,11 @@ class Place(BaseModel):
 
 
 class GoogleMapsTextSearchReader(BaseReader):
-    def __init__(self, api_key: Optional[str] = None):
+    def __init__(
+        self,
+        api_key: Optional[str] = None,
+        length_limit: Optional[int] = None,
```

Comment:
did we end up using this param?

Reply:
Thanks for noticing @nerdai . It won't be needed anymore. PR is updated accordingly.